application/app.py

```diff
@@ -266,8 +266,10 @@ def play_card():
     discarded = false_true(request.args.get('discarded'))
     for_wonder = false_true(request.args.get('for_wonder'))
     from_discard_pile = false_true(request.args.get('from_discard_pile'))
+    #trade = false_true(request.args.get('trade'))
```

Comment:
Do we need this to be commented out? Maybe just add the default to the endpoints in the frontend